.gitignore

```diff
@@ -39,3 +39,4 @@ win32/override/README.txt
 *.patch
 *.so
 *~
+.idea
```

Comment:
Unrelated, please remove.

Reply:
Ok, I removed this. This was added to ignored list for cLion IDE. What IDE do you use to debug this project?